sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-files/ece-response-file-detail.ts

```diff
@@ -57,8 +57,7 @@ export class ECEResponseFileDetail extends ECEResponseFileRecord {
    */
   get payToSchoolAmount(): number {
     const schoolAmount = this.line.substring(144, 148);
-    const schoolAmountDecimal = this.line.substring(148, 150);
-    return +`${schoolAmount}.${schoolAmountDecimal}`;
+    return +`${schoolAmount}`;
```

Comment:
This should be adjusted as one of the below options. Having the string interpolation `${schoolAmount}` does not make much sense here.
```ts
return +this.line.substring(144, 148);
```
or

```ts
return +schoolAmount;
```